### DISCUSSION TITLE
Nuxt refresh token fallback strategies

### DISCUSSION THREAD

**Discussion by vladddbbb** [nuxt/nuxt#16946] [Questions]
Hello! I want to implement fallback strategy for token expiration. My thoughts are about creating some global interval, which checks existing token once per N minutes and refreshes it when expired. It'd be useful when user interactions 'idling' and server requests wont have been sending long time (e.g. large form filling and etc.), so the question is: how to create global interval for whole app, when all entripoints are distributed? (App has many non-intersectable pages, but should be linked by refresh fallback function). Thanks in advance!

**Comment by TheAlexLichter** [comment]
There are multiple options but to me, a composable (which you then can trigger e.g. in a plugin or in your `App.vue`) sounds like a valid strategy to me.
